app/views/comments/_comment_proper.html.erb

```diff
@@ -10,16 +10,17 @@
     <div class="inner-comment">
       <% if comment.decorate.low_quality %>
         <div class="low-quality-comment-marker">
-        <%= image_tag(cl_image_path(asset_path("/assets/sloan.png"),
-          type: "fetch",
-          width: 50,
-          height: 50,
-          crop: "imagga_scale",
-          quality: "auto",
-          flags: "progressive",
-          fetch_format: "auto",
-          sign_url: true), class: "sloan", alt: "sloth-mascot") %>
-          Comment marked as low quality/non-constructive by the community <a href="/code-of-conduct">View code of conduct</a>
+          <%= image_tag(cl_image_path(asset_path("/assets/sloan.png"),
```

Comment:
what happens if you indent cl_image_path on the next line?

Reply:
If I indent `cl_image_path` I would need to change that block to this to avoid linting errors, this might indeed be better:

```ruby
          <%= image_tag(
                cl_image_path(asset_path("/assets/sloan.png"),
                              type: "fetch",
                              width: 50,
                              height: 50,
                              crop: "imagga_scale",
                              quality: "auto",
                              flags: "progressive",
                              fetch_format: "auto",
                              sign_url: true),
                class: "sloan",
                alt: "sloth-mascot",
              ) %>
```

Reply:
Ok got it, I'd leave it like it is then :D